packages/open-truss/src/configuration/RenderConfig.tsx

```diff
@@ -19,19 +20,30 @@ export interface OpenTrussComponentExports {
 export type COMPONENTS =
   | Record<string, OpenTrussComponent>
   | Record<string, OpenTrussComponentExports>
+export type FrameWrapper = React.FC<
+  React.PropsWithChildren<{ frame: FrameType; configPath: string }>
+>
 
 export function RenderConfig({
   components: appComponents,
   config,
+  FrameWrapper,
 }: {
   components: COMPONENTS
   config: string
+  FrameWrapper?: FrameWrapper
```

Comment:
What do you think about making FrameWrapper into an OT component, adding a frame wrapper annotation to the config, and teaching the renderer to handle this? 

Over time I get the feeling that we'll be adding a lot of options to the renderer and instead of passing them through like we're doing here I wonder if we can extend the config to handle this. For now, this perhaps could be at the workflow level such as:

```
workflow:
  version: 1
  frameWrapper: ConfigBuilderFrameWrapper
  frames:
...
```

Maybe eventually we can create a global OT config where this could be set globally and perhaps group level configs. But a problem for another time :).

Reply:
whoa yes this could be cool and really powerful! Lemme take a look

Reply:
done in https://github.com/open-truss/open-truss/pull/101/commits/18ae1f0af75462d3f7779a85df2338fe9ae446cb 🎉 this is very cool I think